gitlab-pipeline/stage/build.yml

```diff
@@ -2,8 +2,7 @@
 build_client:
   only:
     variables:
-      - $BUILD_CLIENT == "true"
-      - $RUN_INTEGRATION_TESTS == "true"
+      - ( $BUILD_CLIENT == "true" && $RUN_INTEGRATION_TESTS == "true" )
```

Comment:
I think this is wrong, it should simply be `$BUILD_CLIENT == "true"`.

For example, when we do the special trigger to publish Docker client images, we use BUILD_CLIENT true but RUN_INTEGRATION_TESTS false. And generally speaking at this stage you don't care if you have integ tests or not, just follow BUILD_CLIENT

Reply:
Hmm, yes, if I understand you correctly, remove `RUN_INTEGRATION_TESTS` completely?

Reason I kept it was that it was just there, but all the same. Removing?

Reply:
> Hmm, yes, if I understand you correctly, remove `RUN_INTEGRATION_TESTS` completely?

Yes

> Reason I kept it was that it was just there, but all the same. Removing?

Not really. The detail here is that
```
- $BUILD_CLIENT == "true"
- $RUN_INTEGRATION_TESTS == "true"
```
means `$BUILD_CLIENT == "true" || $RUN_INTEGRATION_TESTS == "true"`

Reply:
And it had a purpose before, but not anymore with this PR.